e2e/backend.go

```diff
@@ -18,6 +18,7 @@ type IntegrationBackend interface {
 // BackendFake implements IntegrationBackend interface
 type BackendFake struct {
 	nodes           []string
+	votingPowerMap  map[pbft.NodeID]uint64
```

Comment:
You may need https://github.com/0xPolygon/pbft-consensus/pull/68/files changes. It seems we can close(if you take all of this changes)/merge it there.

Reply:
Eventually left this part, as I've talked with @Nemanja0x and seems like he don't need `VotingPower` exposed in the `ValidatorSet` interface. Also, as it is unneeded by consensus library, therefore won't expand it.